src/content/posts/2023-07-12-astro-redirects.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Creating A Redirect in Astro"
+slug: creating-a-redirect-in-astro
+date: 2023-07-14 11:27:06
+tags: 
+- frontend
+- Astro
+- SSG
+categories:
+- frontend
+- css
+twitter_text: "Creating A Redirect in Astro"
+authors: 
+- Ted Krueger
+image: https://images.unsplash.com/photo-1516979187457-637abb4f9353?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770&q=80
+image_url: https://unsplash.com/photos/eMP4sYPJ9x0
+image_credit: Alexander Grey
+---
+
+We needed to redirect a current url to a different page. In our case, we wanted `/agenda` to redirect to an event page. So, to explain it like you're a five year old, who knows a decent amount of developer knowledge, when someone would go to `website-name.com/agenda` they would _actually_ go to `website-name.com/desired-page`.
+
+Now it looks like you _used_ to be able to do this, and still might depending on your version of Astro. The [Astro.redirect](https://docs.astro.build/en/reference/errors/static-redirect-not-available/) method is depricated as of version 2.6. No worries though. I'll be showing you a different method that Astro reccomends using the `meta refresh attribute`.
+
+Before I get into the nuts and bolts of the post I must admit that my attention to detail in the Astro docs was pretty lacking because I missed them suggest the use of the meta refresh. I did, however, see this in [Lyod Atkinson's post](https://www.lloydatkinson.net/posts/2022/static-site-redirects-with-astro/) on his blog. In his post, Atkinson shows us how he uses the meta refresh attribute to redirect to his latest blog post. In our case we wanted our redirect to be specific so we didn't need to worry about some of the extra stuff Lloyd was doing.
```

Comment:
```suggestion
Before I get into the nuts and bolts of the post, I must admit that my attention to detail in the Astro docs was pretty lacking because I missed them suggesting the use of the meta refresh. I did, however, see this in [Lloyd Atkinson's post](https://www.lloydatkinson.net/posts/2022/static-site-redirects-with-astro/) on his blog. In his post, Atkinson shows us how he uses the meta refresh attribute to redirect to his latest blog post. In our case, we wanted our redirect to be specific so we didn't need to worry about some of the extra stuff Lloyd was doing.
```